apps/chatter/consumers.py

```diff
@@ -4,8 +4,11 @@
 from rest_framework.authtoken.models import Token
 from django.contrib.auth.models import AnonymousUser
 from channels.db import database_sync_to_async
-from .models import Message
+from .models import Entry
 from .models import Room
+from asgiref.sync import sync_to_async
```

Comment:
Please follow pep8 style guide (I'll look into adding linting to django-api-starter) https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Am I STL making this mistake?
S tandard libraries
T hird party imports
L ocal project imports

There, remembered forever now :D